manifests/init.pp

```diff
@@ -990,7 +990,7 @@
             require => Package[$my_package_name],
           }
 
-          file { 'pam_common_noninteractive_session':
+          file { 'pam_common_session_noninteractive':
```

Comment:
Yes, it removes the logic that has to handle that file as a special case. See https://github.com/emahags/puppet-module-pam/commit/b0b56684c24e2e891e2bb50db23a00dac42956f0